src/routes/controllers/articles.controller.js

```diff
@@ -48,12 +49,21 @@ export async function createArticle(req, res) {
       ).toLowerCase(),
       description,
       body,
-      imageList: images,
-      tagList: tags,
       readTime: timeToReadArticle,
       subscriptionType: FREE,
       status: DRAFT,
     });
+    if (tags) {
+      console.log(tags);
+      const createdTag = await createTag(tags, createdArticle.id);
+      if (!createdTag) {
+        return errorResponse(
+          res,
+          415,
```

Comment:
Could you clarify what this piece of code is doing exactly? What i understand is that you are calling a wrapper around ```Tag.create()``` right? Is the error you are returning to tell the server to send an array in the request body next time he is calling the endpoint or you are telling the user that you were expecting an ```array``` from the server but it returned a ```string```?

Reply:
it a method that returns an array if tags were created.
it fails only when the tags passed into it is a string expects: ['tag1', 'tag2'...] but got 'tags1, tag2,...'.